libs/langgraph/src/pregel/algo.ts

```diff
@@ -356,22 +419,33 @@ export function _prepareNextTasks<
               runName: packet.node,
               callbacks: manager?.getChild(`graph:step:${step}`),
               configurable: {
-                [CONFIG_KEY_SEND]: _localWrite.bind(
-                  undefined,
-                  (items: [keyof Cc, unknown][]) => writes.push(...items),
-                  processes,
-                  channels
-                ),
-                [CONFIG_KEY_READ]: _localRead.bind(
-                  undefined,
-                  checkpoint,
-                  channels,
-                  {
-                    name: packet.node,
-                    writes: writes as Array<[string, unknown]>,
-                    triggers,
-                  }
-                ),
+                // eslint-disable-next-line @typescript-eslint/no-explicit-any
+                [CONFIG_KEY_SEND]: (writes_: [string, any][]) =>
+                  _localWrite(
+                    step,
+                    (items: [keyof Cc, unknown][]) => writes.push(...items),
+                    processes,
+                    channels,
+                    managed,
+                    writes_
+                  ),
+                [CONFIG_KEY_READ]: (
+                  select_: Array<keyof Cc> | keyof Cc,
+                  fresh_: boolean = false
```

Comment:
Why are these params suffixed?

Also what do they represent?

Reply:
suffix to show we shouldn't do anything with them besides passing args through. They're the two args which we aren't populating, but this func requires so we create a `partial` func. No change from what we had before, except a new type error occurred with the existing implementation. (plus I think naming conflicts)